core/symbol/downcase_spec.rb

```diff
@@ -20,6 +20,13 @@
     end
   end
 
+  ruby_version_is '2.4' do
+    it "uncapitalizes all characters regardless of if it is Unicode" do
```

Comment:
Same as above, I think it would read better as
`uncapitalizes all Unicode characters`, could you change it? (and below as well)